Corona-Warn-App/src/main/res/values/colors.xml

```diff
@@ -122,4 +123,6 @@
     <color name="admissionTileBackground">@color/colorPersonOverviewQrCardGrey</color>
     <color name="contactDiaryOverviewUnClickableColor">#FFFFFF</color>
     <color name="contactDiaryOverviewBorderColor">#E2E2E2</color>
+    <color name="maskGBadgeColor">#1F5932</color>
+    <color name="mask_badge_color">#505968</color>
```

Comment:
Maybe rename these to:
1. `maskGreenBadgeColor`
2. `maskBadgeColor` so it won't be the only value with underscores from the color values.

Reply:
Renamed , G is not for Green. it is for GStatus